pkg/gatewayserver/io/ws/ws_test.go

```diff
@@ -34,11 +34,11 @@ import (
 	"go.thethings.network/lorawan-stack/v3/pkg/encoding/lorawan"
 	"go.thethings.network/lorawan-stack/v3/pkg/frequencyplans"
 	"go.thethings.network/lorawan-stack/v3/pkg/gatewayserver/io"
-	. "go.thethings.network/lorawan-stack/v3/pkg/gatewayserver/io/basicstationlns"
-	"go.thethings.network/lorawan-stack/v3/pkg/gatewayserver/io/basicstationlns/messages"
 	"go.thethings.network/lorawan-stack/v3/pkg/gatewayserver/io/mock"
+	. "go.thethings.network/lorawan-stack/v3/pkg/gatewayserver/io/ws"
+	"go.thethings.network/lorawan-stack/v3/pkg/gatewayserver/io/ws/lbslns"
 	"go.thethings.network/lorawan-stack/v3/pkg/log"
-	pfconfig "go.thethings.network/lorawan-stack/v3/pkg/pfconfig/basicstationlns"
+	pfconfig "go.thethings.network/lorawan-stack/v3/pkg/pfconfig/lbslns"
```

Comment:
Is it feasible to not test `ws` with `lbslns`?

Reply:
That would require generating mock messages in the `lbslns` package and using only a raw byte stream in `ws`. Not sure if it's worth the effort.

Reply:
OK